addons/rhs/CfgMagazineWells.hpp

```diff
@@ -32,5 +32,9 @@ class CfgMagazineWells
    class TB_magwell_33Rnd_9x19
    {
        TB_33Rnd_9x19_Glock18[] = {"TB_mag_33Rnd_9x19_FMJ","TB_mag_33Rnd_9x19_JHP"};
-   }; 
+   };
+   class TB_magwell_MPR_RHS
+    {
+        TB_magwell_MPR_RHS[] = {"TB_mag_MPR_10Rnd_Slug"};
```

Comment:
@E-for-Eron geht das? muss nicht +=?

Reply:
Hatte beides beim Test funktioniert. Prüfe es um 21Uhr und gebe dann grünes Licht.

Reply:
Geht auch ohne += . Von der Syntax klingt deins aber eigentlich richtig. Pack es dir rein, wenn du damit besser schlafen kannst :) Beide Versionen klappen.